keyboards/ergodox_infinity/keymaps/nordic_ergo/keymap.c

```diff
@@ -1,24 +1,16 @@
 // Nordic layout for Ergodox infinity
-
 #include QMK_KEYBOARD_H
-#include "debug.h"
 #include "action_layer.h"
 #include "version.h"
 #include "keymap_nordic.h"
 #include "keymap_german.h"
 
-// Layer names
-#define BASE 0 // default layer
-#define FUNCL 1 // function layer
-#define SYMB 2 // symbol layer
-#define MDIA 3 // media keys
+#define BASE 0   // default layer
+#define FUNCL 1  // function layer
+#define SYMB 2   // symbol layer
+#define MDIA 3   // media keys
 
-enum custom_keycodes {
-    PLACEHOLDER = SAFE_RANGE, // can always be here
-    EPRM,
-    VRSN,
-    RGB_SLD
-};
+enum custom_keycodes { PLACEHOLDER = SAFE_RANGE, EPRM, VRSN, RGB_SLD };
```

Comment:
This should remain as multi-line for readability.

Also, the `PLACEHOLDER` and `EPRM` keycodes aren't needed, as they aren't being used. `VRSN` technically isn't needed either, but it's a good example to have.

```c
enum custom_keycodes {
    VRSN = SAFE_RANGE,
    RGB_SLD
};

```